eng/Versions.props

```diff
@@ -1,5 +1,5 @@
 <Project>
```

Comment:
Everything in this file is the same except:

1. I added labels to the PropertyGroups. DARC seems to find the Automated label and add new dependencies to that group.
2. I added the MicrosoftAspNetCoreAppRefVersion and MicrosoftNETCoreAppRuntimewinx64Version property versions via DARC.

Otherwise, everything else has been preserved but rearranged according to the type of dependency it is.